src/containers/MapConfig.js

```diff
@@ -0,0 +1,8 @@
+export const mapOptions = {
```

Comment:
object name should follow file name. Let's just go with "config" as the file and object name.